dmalloc_argv.c

```diff
@@ -2223,12 +2215,6 @@ static	void	do_arg(argv_t *grid, argv_t *match_p, const char *close_p,
       *okay_bp = ARGV_FALSE;
     }
   }
-  else if (argv_close_enable_b && close_p != NULL) {
```

Comment:
Why was this removed?  Was in unreachable?

Reply:
Yes, because line 2214/2206 already check it.